### PR TITLE
Feature/vtn standards verify engines

### PR DESCRIPTION
**Pull request by howardlei82**

### CODE REVIEW

File: src/docs/developer/engines/cognitive/README.md [hunk 1]
```diff
@@ -31,10 +31,9 @@ Class | Capability | Description
 Audio | [Audio Fingerprinting](/developer/engines/cognitive/audio/audio-fingerprinting/) | Recognizes a specific audio segment, eg. a radio advertisement, as it appears in a longer audio file or on its own.
 Biometrics | [Face Detection](/developer/engines/cognitive/biometrics/face-detection/) | Detects the presence of one or multiple faces in an image or video.
 Biometrics | [Face Recognition](/developer/engines/cognitive/biometrics/face-recognition/) | Identifies one or multiple people in an image or video by associating each individual's face to their name.
-Biometrics | [Face Verification](/developer/engines/cognitive/biometrics/face-verification/) | Determines the similarity between the face in an image to the face of a specified username. In `enroll` mode, the engine enrolls the face image into the library under the username.
-Biometrics | [Speaker Verification](/developer/engines/cognitive/biometrics/speaker-verification/) | Determines the similarity between the speaker's voice in an audio file to the voice of a person with a specified username. In `enroll` mode, the engine enrolls the speaker's voice into the library under the username.
```
Comment (albertcbrown) [inline src/docs/developer/engines/cognitive/README.md:35]: Why was speaker verification removed?

File: src/docs/developer/engines/cognitive/facial-features/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The facial features engine performs [segment processing](/developer/engines/processing-modes/segment-processing/).
+
+It accepts as input a custom binary file containing the following in the respective order:
+
+    1. 8 bytes containing the number of bytes of a byte-encrypted json string
+    2. A byte-encrypted JSON string
+    3. 8 bytes containing the number of bytes of a binary audio file
+    4. Binary audio file
+
```
Comment (albertcbrown) [inline src/docs/developer/engines/cognitive/facial-features/README.md:47]: I would also suggest it takes in:
1. Frames
2. Video

Comment (albertcbrown) [reply src/docs/developer/engines/cognitive/facial-features/README.md:47]: We should also give an example

File: src/docs/developer/engines/cognitive/facial-features/taskOutput.example.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "schemaId": "https://docs.veritone.com/schemas/vtn-standard/master.json",
+  "validationContracts": [
+    "facialFeatures"
+  ],
+  "object": [
+    {
+      "type": "facial-features",
```
Comment (albertcbrown) [inline src/docs/developer/engines/cognitive/facial-features/taskOutput.example.json:8]: I would include an example of an object in a time series.

File: src/docs/developer/engines/cognitive/verification/speaker-verification/README.md [hunk 4]
```diff
@@ -38,12 +38,43 @@ Training for the Speaker Verification Engine is done by using the `enroll` mode
...
+        "dbUser": "postgres",
+        "dbHost": "127.0.0.1",
+        "dbDatabase": "postgresdb",
+        "dbSchema": "public",
+        "dbPort": 5432,
+        "userPhrase": "hello world",
+        "veritoneToken": "39ffd7a5-e962-4632-87c0-4d1616c7a9f6",
+        "veritoneApiBaseUrl": "https://api.veritone.com/v3/graphql",
+    }
+
```
Comment (albertcbrown) [inline src/docs/developer/engines/cognitive/verification/speaker-verification/README.md:63]: This is too specific to us.  The input should be everything it needs to do its job without having to lookup anything.

Comment (albertcbrown) [reply src/docs/developer/engines/cognitive/verification/speaker-verification/README.md:63]: Typically this is an input file (audio or video) with a libraryID.

File: src/docs/developer/engines/cognitive/verification/speaker-verification/taskOutput.example.json [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
+  "object": [{
+    "type": "speaker-verification",
     "mode": "verify",
     "entityId": "11a14999-0531-4d3e-9a44-68cdd4f93659",
     "libraryId": "13e6f4a3-0d5c-4e11-9a30-913e981cb9ad",
-    "confidence": 0.80
-  }
-}
\ No newline at end of file
+    "confidence": 0.80,
```
Comment (albertcbrown) [inline src/docs/developer/engines/cognitive/verification/speaker-verification/taskOutput.example.json:11]: Should we have pass/maybe/no?

File: src/docs/developer/engines/cognitive/facial-features/README.md (hunk 2 above)
Comment (howardlei82) [reply src/docs/developer/engines/cognitive/facial-features/README.md:49]: Updating the README with an example of the JSON string. 

Regarding taking in frames/video, that's not currently what the engine is capable of doing, but added a note in the README saying that's a capability we'll try to include in the future. Would this be okay as a version 1?

File: src/docs/developer/engines/cognitive/facial-features/taskOutput.example.json (hunk 3 above)
Comment (howardlei82) [reply src/docs/developer/engines/cognitive/facial-features/taskOutput.example.json:8]: The engine processes the input face landmark time-series, and outputs a single confidence score for both lipVoiceCorrelation and lipMovement with respect to the input time series. 

Hence, the output object is currently not a time series, but we can think of ways to make it a time-series in future versions of the engine.

File: src/docs/developer/engines/cognitive/verification/speaker-verification/README.md (hunk 4 above)
Comment (howardlei82) [reply src/docs/developer/engines/cognitive/verification/speaker-verification/README.md:61]: Removed the fields veritoneToken and veritoneApiBaseUrl in the object.

File: src/docs/developer/engines/cognitive/verification/speaker-verification/taskOutput.example.json (hunk 5 above)
Comment (howardlei82) [reply src/docs/developer/engines/cognitive/verification/speaker-verification/taskOutput.example.json:11]: Currently the engine outputs the raw confidence score, and it's up to the app to compare the score to a threshold to determine pass/maybe/no. The threshold is tied to the needs of the app's business logic. If we want the engine to output pass/maybe/no, we would need to pass the threshold as an additional input to the engine. Let me know if it's okay to leave it as is.

**Pull request merged**